component/prometheus/exporter/windows/config.go

```diff
@@ -0,0 +1,190 @@
+package windows
+
+import (
+	windows_integration "github.com/grafana/agent/pkg/integrations/windows_exporter"
+)
+
+// DefaultArguments holds non-zero default options for Arguments when it is
+// unmarshaled from YAML.
+//
+// Some defaults are populated from init functions in the github.com/grafana/agent/pkg/integrations/node_exporter package.
+var DefaultArguments = Arguments{
+	EnabledCollectors: "cpu,cs,logical_disk,net,os,service,system",
+}
```

Comment:
I think this is probably an opportunity to turn EnabledCollectors into a `[]string` on the Flow side so it's less awkward from a configuration perspective: 

```
prometheus.exporter.windows "example" {
  enabled_collectors = ["cpu", "cs", "logical_disk", ...] 
}
```

What do you think?

Reply:
I hope in future, river will have some additional functions like [`yamldecode`](https://developer.hashicorp.com/terraform/language/functions/yamldecode) to allow more flexibility on complex types.